src/Regulator.php

```diff
@@ -214,7 +217,7 @@ protected function dispatch_callback( Task $task, int $delay ): void {
 			DB::beginTransaction();
 
 			if ( Action_Scheduler_Methods::has_scheduled_action( $this->process_task_hook, [ $args_hash ], $group ) ) {
-				throw new ShepherdTaskAlreadyExistsException( 'The task is already scheduled.' );
+				throw new ShepherdTaskAlreadyExistsException( esc_html__( 'The task is already scheduled.', 'stellarwp-shepherd' ) );
```

Comment:
Here and elsewhere, use `esc_html_x` to provide the translator an idea of what type of string this is.
Translators will not get the code around it as context, just the string itself and the context you provide for it.